networking/eth2/src/integration-test/java/tech/pegasys/teku/networking/eth2/BeaconBlocksByRangeIntegrationTest.java

```diff
@@ -155,38 +156,53 @@ void requestBlockBySlot_shouldReturnEmptyWhenBlockIsNotKnown() throws Exception
     assertThat(result).isEmpty();
   }
 
-  @ParameterizedTest(name = "enableAltairLocally={0}, enableAltairRemotely={1}")
-  @MethodSource("altairVersioningOptions")
-  public void requestBlockBySlot_withDisparateVersionsEnabled_requestPhase0Blocks(
-      final boolean enableAltairLocally, final boolean enableAltairRemotely) throws Exception {
-    final Eth2Peer peer = createPeer(enableAltairLocally, enableAltairRemotely);
+  @ParameterizedTest(name = "{0} => {1}, nextSpecEnabledLocally={2}, nextSpecEnabledRemotely={3}")
+  @MethodSource("generateSpecTransitionWithCombinationParams")
+  public void requestBlockBySlot_withDisparateVersionsEnabled_requestBaseMilestoneBlocks(
```

Comment:
as both blocks are from baseMilestone, deneb test is lost now, isn't it?

Reply:
yes, deneb it is actually tested only on the other test `requestNextSpecBlocks` when both peers are in `nextSpec` enabled

Reply:
it should be good enough to me